log/log.go

```diff
@@ -87,13 +90,13 @@ func StartObserver(o Options) error {
 	if o.LogPath != "none" {
 		if o.LogFilter == "all" || o.LogFilter == "policy" {
 			// watch alerts
-			go logClient.WatchAlerts(o.LogPath, o.JSON)
+			go logClient.WatchAlerts(o.LogPath, o.JSON, o.Namespace, o.LogType, o.Operation)
```

Comment:
```suggestion
			go logClient.WatchAlerts(o)
```
We should prolly now pass the entire option struct instead of manually expanding the entire options struct as function object. What do you think?

Reply:
yeah, we can do that ... looks much more cleaner